content/lessons/testing/testing-workshop.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Testing Workshop
+template: lesson
+draft: false
+slug: /courses/Testing/testing-workshop
+course: Testing
+tags:
+  - Jest
+description: "If you've made to this section, you likely already have a good grasp on frontend testing. This workshop will challenge you to hone your skills with an open ended approach. The workshop features a simple app implemented in a few different ways. You job is to provide tests for each app. We've provided 'tickets' with testing directions to keep you on track. Check out the video below for more information"
```

Comment:
`If you've made **it** to this section...`

Reply:
Nice catch, fixed!